nullaway/src/main/java/com/uber/nullaway/handlers/contract/fieldcontract/EnsuresNonNullHandler.java

```diff
@@ -85,11 +86,21 @@ protected boolean validateAnnotationSemantics(
             .stream()
             .map(e -> e.getSimpleName().toString())
             .collect(Collectors.toSet());
+    Set<String> nonnullStaticFieldsOfReceiverAtExit =
+        analysis
+            .getNullnessAnalysis(state)
+            .getNonnullStaticFieldsAtExit(new TreePath(state.getPath(), tree), state.context)
+            .stream()
+            .map(e -> e.getSimpleName().toString())
+            .collect(Collectors.toSet());
+    nonnullFieldsOfReceiverAtExit.addAll(nonnullStaticFieldsOfReceiverAtExit);
```

Comment:
```suggestion
    nonnullFieldsAtExit.addAll(nonnullStaticFieldsOfReceiverAtExit);
```
They are not just on the receiver anymore

Reply:
Fixed!